cluster-http/src/test/scala/akka/cluster/bootstrap/contactpoint/HttpContactPointRoutesSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster.bootstrap.contactpoint
+
+import akka.cluster.{ Cluster, ClusterEvent }
+import akka.cluster.ClusterEvent.ClusterDomainEvent
+import akka.cluster.bootstrap.ClusterBootstrapSettings
+import akka.cluster.http.management.ClusterHttpManagementJsonProtocol
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.testkit.TestProbe
+import org.scalatest.{ Matchers, WordSpecLike }
+
+class HttpContactPointRoutesSpec
+    extends WordSpecLike
+    with Matchers
+    with ScalatestRouteTest
+    with HttpBootstrapJsonProtocol {
+
+  "Http Bootstrap routes" should {
+
+    val settings = ClusterBootstrapSettings(system.settings.config)
+    val httpBootstrap = new HttpClusterBootstrapRoutes(settings)
+
+    "empty list if node is not part of a cluster" in {
+      ClusterBootstrapRequests.bootstrapSeedNodes("") ~> httpBootstrap.routes ~> check {
+        responseAs[String] should include(""""seedNodes":[]""")
+      }
+    }
+
+    "include seed nodes when part of a cluster" in {
+      val cluster = Cluster(system)
+      cluster.join(cluster.selfAddress)
+
+      val p = TestProbe()
+      cluster.subscribe(p.ref, ClusterEvent.InitialStateAsEvents, classOf[ClusterDomainEvent])
+      val up = p.expectMsgType[ClusterEvent.MemberUp]
```

Comment:
when subscribing to all ClusterDomainEvent I don't think it's guaranteed that MemberUp will be the first, change the `subscribe` to only `MemberUp`